server/index.js

```diff
@@ -0,0 +1,7 @@
+const express = require("express");
+const app = express();
+const port = 3001;
```

Comment:
environment variable :)

Reply:
@piyushagwl007 has taken care of this.

Reply:
Yes. I will be taking care of this in my subsequent PR.